### ISSUE TITLE
I'm unable to create a project

### ISSUE THREAD

**Issue by CamburgerHelper** [chrismaltby/gb-studio#299]
After downloading GB Studio I am unable to create a project. Any time I try to create one the "creating" icon pops up but nothing happens. I've tried using both the 64 bit and 32 bit versions. I've also tried running it as an administrator and that didn't work either. 

**To Reproduce**
1. Download Windows 64-bit Squirrel Installer
2. Extract All
3. Click on "GB-Studio-1.1.0 Setup"
4. Click on "Run"
5. Create new project

**Expected behavior**
The "creating" icon appears and then nothing happens. The software doesn't crash or anything, it just does absolutely nothing.

**Screenshots**
![image](https://user-images.githubusercontent.com/56135981/66161521-79d38200-e5fa-11e9-81bb-87cc9ae227ff.png)

**Platform (please complete the following information):**
 - OS: Windows 10 ver. 1903
 - App version 1.1.0

**Comment by CamburgerHelper** [comment]
Never mind! It was an issue on my end that I was able to resolve.

**Comment by RichardULZ** [comment]
What was the fix? Would be nice to know if someone stumbles upon the same issue.

**Comment by locherunderscore** [comment]
Maybe he updated his drivers + let his windows update do its work. We may never know..

**Comment by CamburgerHelper** [comment]
Hey, sorry everybody! I realize I should have probably left this on a less vague note. What had happened was that my antivirus (Bitdefender) thought it was malware or something and blocked it without letting me know. What I had to do was go in and make GBStudio and exception for ransomware remediation and online threat prevention. I also added it to my list of safe files. So in the end it was pretty simple and should've been a little more obvious to me in the first place lol
